src/main/java/com/wolox/training/controllers/BookController.java

```diff
@@ -19,28 +23,19 @@
  */
 @RestController
 @RequestMapping("/api/books")
+@Api
 public class BookController {
 
     @Autowired
     private BookRepository bookRepository;
 
-    /**
-     * This method returns views concatenating "Hello" + the name that we send by parameter
-     * @param name: The name to be concatenated in the views
-     * @param model
-     * @return views named greeting.html
-     */
-    @GetMapping("/greeting")
-    public String greeting(@RequestParam(name="name", required=false, defaultValue="World") String name, Model model) {
-        model.addAttribute("name", name);
-        return "greeting";
-    }
-
     /**
      * This method retrieve all books from database and showing them in json format
      * @return Iterable<Book> object to list all books
      */
     @GetMapping
+    @ApiOperation(value = "Return all books", response = Book.class)
```

Comment:
You could add the following in order to specify that you will be returning a list of Books. 
```suggestion
    @ApiOperation(value = "Return all books", response = Book.class, multiValuedResponse=true)
```
or
```suggestion
    @ApiOperation(value = "Return all books", response = Book.class, responseContainer="List")
```
I don't remember which one is correct due to your swagger version 😋